sema/commons/clean/clean.py

```
@@ -166,7 +166,7 @@ def build_clean_chain(*specs) -> Callable:
     log.debug(f"chain of funtions {chain_fn=}")
     # group per level
     grouped_fn = reduce(  # group chain of cleaners per level
-        lambda d, fn: d.get(fn.level, list()).append(fn) or d,
+        lambda d, fn: d.get(fn.level, list()).append(fn) or d,  # type: ignore
```

Ruff

Unnecessary `list` call (rewrite as a literal)

Rewrite as a literal

(C408)